db/schema.rb

```diff
@@ -35,6 +33,17 @@
     t.index ['post_id'], name: 'index_likes_on_post_id'
   end
 
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.string 'text'
```

Comment:
- [ ]  like the previous reviewer noted, You are required to create all tables for schema.rb using migrations. your migration files dont relate to your schema. For example, you can create one for the users table by entering something similar to this in the terminal.
rails g migration Users name:string photo:string posts_counter:integer

it creates something like this in the migration file
```
create_table :users do |t|
    t.string :name
    t.string :photo 
    t.text :bio
    t.integer :posts_counter
    t.timestamps # this will crate uopdated_at and created_at columns
  end

```
you den `db migrate` it to update your schema.

see more [here](https://guides.rubyonrails.org/active_record_migrations.html)